cit-api/pipeline/management/commands/bootstrap.py

```diff
@@ -79,7 +79,7 @@ def handle(self, *args, **options):
             print(f'Importing {resource.display_name}...')
             import_csv_resources(resource.name)
 
-        calculate_nearest_location_types_outside_50k()
+        # calculate_nearest_location_types_outside_50k()
```

Comment:
Do we want these distance functions commented out?

Reply:
No, this will change next week